Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115.py

```diff
@@ -91,10 +91,23 @@ def incident_add_journal_entry_request(self, id_: str | None, author, notes: str
 
     def incident_list_alerts_request(self, page_size: str | None, page_number: str | None, id_: str | None) -> dict:
         params = assign_params(pageNumber=page_number, pageSize=page_size)
```

Comment:
Did they change the API endpoint here? I ask because it looks like we are no longer using the page_number and page_size arguments. If the new endpoint supports it, we should try and keep this functionality.

Reply:
Sounds good. Let me know if there is anything additional you need from us.

Currently we are unable to change the pathing of the pack, however by changing the `name` field in the pack metadata file, it will change the display name for the pack. Additionally, we would also need to update the svg logo for the integrations as well since the pack uses these images.

https://github.com/demisto/content/blob/master/Packs/RSANetWitness_v11_1/pack_metadata.json#L2 - Changing the name here will change the name of the pack.

https://github.com/demisto/content/blob/master/Packs/RSANetWitness_v11_1/Integrations/RSANetWitness_v11_1/RSANetWitness_v11_1_dark.svg - For this integration specifically, here is one of the logos.

https://github.com/demisto/content/blob/master/Packs/RSANetWitness_v11_1/Integrations/RSANetWitnessv115/RSANetWitnessv115_dark.svg - For the other integration in the pack, here is one of the other logos.

@ShahafBenYakir FYI